src/cephadm/cephadm

```diff
@@ -4324,17 +4406,20 @@ class AgentGatherer(Thread):
                 try:
                     self.data, change = self.func()
                 except Exception as e:
-                    logger.error(f'{self.gatherer_type} Gatherer encountered exception gathering data: {e}')
+                    logger.error(
+                        f'{self.gatherer_type} Gatherer encountered exception gathering data: {e}')
                     self.data = None
                 if ack != self.ack or change:
                     self.ack = ack
                     self.agent.wakeup()
 
                 end_time = time.monotonic()
                 run_time = datetime.timedelta(seconds=(end_time - start_time))
-                self.recent_iteration_run_times[self.recent_iteration_index] = run_time.total_seconds()
+                self.recent_iteration_run_times[self.recent_iteration_index] = run_time.total_seconds(
+                )
```

Comment:
is there a better way we can split up this line than a single right paren on its own line?